app/react/docker/networks/queries/useConnectContainer.ts

```diff
@@ -11,6 +10,7 @@ import {
 } from '@/react-tools/react-query';
 
 import { queryKeys as dockerQueryKeys } from '../../queries/utils';
+import { addNodeName } from '../../proxy/addNodeName';
```

Comment:
We are adding node header. `addNodeName` does seem a bit less described to me. `addNodeHeader` so something

Reply:
Nice comment, will update this existing function name